org.aposin.licensescout.core/src/main/java/org/aposin/licensescout/license/ArtifactServerUtil.java

```diff
@@ -31,9 +31,10 @@
 import org.codehaus.plexus.util.xml.pull.XmlPullParserException;
 
 /**
- * Utility methods related to accessing a server that is caching maven central.
+ * Utility methods related to accessing a server that is caching Maven central.
  * 
- * <p>The instance of this class checks if the server is reachable under the configured URL. 
+ * <p>The instance of this class checks if the server is reachable under the
+ * configured URL.</p>
```

Comment:
Please, could you revert all those changes that are just formatting, to make easier the review?

Reply:
Reverted some, but kept very long lines broken up.